Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -1072,14 +1122,16 @@ def run(request, workspace, extra_cmdline_args):
 
         save_asset_logs = False
         for result in results_per_thread:
+            if result is None:
+                result = Result.Unknown(test_spec=EditorSharedTest)
```

Comment:
Here and for the rest of the same- Should we add some extra information for this Result? Could pass in something useful in the extra_info arg to be printed.